### PR TITLE
docs: update les page

### PR DESCRIPTION
**Pull request by jmcook1186**
Updates les page with additional details, links etc.

### CODE REVIEW

File: docs/_interface/les.md [hunk 1]
```diff
@@ -3,46 +3,60 @@ title: Light client
 sort_key: B
 ---
 
-*Note*: Light client is an experimental feature. Please consider this before using it with large funds.
+Running a full node is the most trustless, private, decentralized and censorship resistant way to interact with Ethereum. It is also the best choice for the health of the network, because a decentralized network relies on having many individual nodes that independently verify the head of the chain. Ina  full node a copy of the blockchain is stored locally enabling users to verify incoming data against a local source of truth. However, running a full node requires a lot of disk space and non-negligible CPU allocation and takes hours (for snap sync) or days (for full sync) to sync the blockchain from genesis. Geth also offers a light mode that overcomes these issues and provides some of the benefits of running a node but requires only a fraction of the resources.
```
Comment (rjl493456442) [inline docs/_interface/les.md:6]: Ina => In a?

File: docs/_interface/les.md [hunk 2]
```diff
@@ -3,46 +3,60 @@ title: Light client
...
+## Light node vs full node
 
-What's the catch you might ask? They are heavily dependant on as of now altruistic light servers. These are full nodes that volunteer to serve data to light clients and which can be easily overwhelmed as the number of clients increase.
+Running Geth in light mode has the following advantages for users:
 
-They also have slightly different security guarantees. Because they don't keep the Ethereum state, they can't validate the blocks in the same way as the full nodes. Instead they fetch block headers and check their Proof-of-Work, assuming the heaviest chain is valid. What this means for you is that if you want to be on the safe side it's advisable to wait for a few blocks worth of confirmations before trusting the validity of a recently mined transaction.
+- Syncing takes minutes rather than hours/days
+- Light mode uses significantly less storage
+- Light mode is lighter on CPU and other resources
+- Light mode is suitable for resource-contrained devices
```
Comment (rjl493456442) [inline docs/_interface/les.md:17]: resource-constrained

File: docs/_interface/les.md [hunk 3]
```diff
@@ -3,46 +3,60 @@ title: Light client
...
 
-By enabling light serving on your full node you'll help the network and light clients. Naturally you might want to cap the amount of resources you dedicate to serving light clients. That's why the same flag that enables this feature also requires you to specify how much resources to dedicate to serving clients. `--light.serve <num>` takes a percentage as value. Number bigger than 100 indicate you want to dedicate more than one thread to this feature, e.g. `--light.serve 200` for 2 cores.
+Light nodes also have different security guarantees than full nodes. Because they don't keep local copies of the Ethereum state, light nodes can't validate the blocks in the same way as the full nodes. Instead they fetch block headers by requestign them from full nodes and check their proof-of-work (PoW), assuming the heaviest chain is valid. This means that it is sensible to wait until a few additional blocks have been confirmed before trusting the validity of a recently-mined transaction.
```
Comment (rjl493456442) [inline docs/_interface/les.md:22]: requestign -> requesting

File: docs/_interface/les.md (hunk 1 above)
Comment (jmcook1186) [reply docs/_interface/les.md]: thanks for catching these! Fixed.

File: docs/_interface/les.md [hunk 4]
```diff
@@ -3,46 +3,60 @@ title: Light client
 sort_key: B
 ---
 
-*Note*: Light client is an experimental feature. Please consider this before using it with large funds.
+Running a full node is the most trustless, private, decentralized and censorship resistant way to interact with Ethereum. It is also the best choice for the health of the network, because a decentralized network relies on having many individual nodes that independently verify the head of the chain. In a  full node a copy of the blockchain is stored locally enabling users to verify incoming data against a local source of truth. However, running a full node requires a lot of disk space and non-negligible CPU allocation and takes hours (for snap sync) or days (for full sync) to sync the blockchain from genesis. Geth also offers a light mode that overcomes these issues and provides some of the benefits of running a node but requires only a fraction of the resources.
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply docs/_interface/les.md]: thanks!

**Review by rjl493456442 — APPROVED** [review]
Just a few nitpicks, otherwise look pretty good to me.

@zsfelfoldi Can you also check it so that we can get it in?

File: docs/_interface/les.md [hunk 5]
```diff
@@ -3,46 +3,60 @@ title: Light client
...
+Data can be requested from this light Geth instance in the same way as for a full node (i.e. using the [JSON-RPC-API](/docs/rpc/server) using tools such as [Curl](https://curl.se/) or Geth's [Javascript console](/docs/interface/javascript-console)). Instead of fetching the data from a local database as in a full node, the light Geth instance requests the data from full-node peers. Transactions can also be sent through the light server. However, light clients are not connected directly to Ethereum Mainnet but a network of light servers that in turn connect to Ethereum Mainnet. This means a transaction submitted by a light client is received first by a light server that then propagates it to full-node peers on Ethereum on the light-client's behalf. This reliance on honest light-servers is one of the trust compromises that comes along with running a light node instead of a full node.
```
[the remark this answers is not in the document]
Comment (jmcook1186) [reply docs/_interface/les.md:47]: yep nice one - fixing now

**Review by s1na — APPROVED** [review]
LGTM

**Pull request merged**